### DISCUSSION TITLE
微信公众号批量下载工具2.5 新增Mac版本

### DISCUSSION THREAD

**Discussion by qiye45** [qiye45/wechatDownload#103] [Q&A] [answered]
2.5 支持了Macos版本🥳；支持表格的实时导出；修复数据采集缺失的问题；修复下载一些公众号失败、导出表格的数据不完整的问题；支持多篇文章下载

![1716128875595](https://github.com/qiye45/wechatDownload/assets/138199658/a419a193-f09c-4df0-b152-550f937ccc17)
![1716128824654](https://github.com/qiye45/wechatDownload/assets/138199658/2c4412ab-da2b-4302-84f5-811993023d93)

<hr /><em>This discussion was created from the release <a href='https://github.com/qiye45/wechatDownload/releases/tag/'>微信公众号批量下载工具2.5 新增Mac版本</a>.</em>

**Comment by 1131161335** [comment]
macOS14.4.1 系统无法安装？

**Reply by qiye45** [reply]
可能要apple芯片的

**Reply by qiye45** [reply, marked as answer]
环境
![image](https://github.com/qiye45/wechatDownload/assets/138199658/5853eb01-8b29-450d-81d7-6b780b9ef902)
![image](https://github.com/qiye45/wechatDownload/assets/138199658/6aa30d7c-6de8-4ad8-b854-f871065abb12)

**Reply by fqh1995** [reply]
我用的2.4版本，没有下载出来文章

**Reply by qiye45** [reply]
看看issue...

**Reply by fqh1995** [reply]
好的，我再试试，2.5 win版本下载99%浏览器提示无法授权下载，有网盘链接吗？

**Reply by fqh1995** [reply]
刚才试了一下，可以下载了，但是排版是乱的（2.4版本）

**Reply by fqh1995** [reply]
你好，工具很给力，就是希望同步给个版本更新网盘链接。已经赞助了一个鸡腿，希望本工具越来越好和帮助更多人

**Reply by qiye45** [reply]
网盘链接：https://pan.quark.cn/s/09bdfa78d09b

**Reply by qiye45** [reply]
感谢打赏🥰

**Reply by fqh1995** [reply]
还有个建议，下载的封面和视频能不能也改成可以自定义增加时间命名，到时候好分类整理？

**Reply by qiye45** [reply]
加到https://github.com/qiye45/wechatDownload/issues/104 啦

**Comment by qhu01** [comment]
<img width="1102" alt="image" src="https://github.com/qiye45/wechatDownload/assets/25233838/8f2793f4-2224-4c7e-bd2d-37b721415bda">
您好！我下载了2.5版本然后用的是M1 Pro的Macbook，但是我看到“获取公众号Id"好像按不了。请问这个正常码？

**Reply by qiye45** [reply]
因为还没适配，可以先试试用链接下载哟
